modules/sdk/gradle-plugins-workspace/src/test/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtensionValidatorTest.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.gradle.plugins.workspace.internal.client.extension;
+
+import com.liferay.gradle.plugins.workspace.internal.client.extension.validator.ClientExtensionValidator;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.nio.file.Files;
+
+import java.util.HashMap;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.Project;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Anderson Luiz
+ */
+public class ClientExtensionValidatorTest {
+
+	@Test
+	public void testShouldValidateSuccessfullyIfNoFrontendTokenDefinitionProvided() {
+		ClientExtension clientExtension = _getThemeCSSClientExtension();
+
+		clientExtension.typeSettings.clear();
+
+		Project project = Mockito.mock(Project.class);
+
+		try {
+			_clientExtensionValidator.validate(clientExtension, project);
+		}
+		catch (GradleException gradleException) {
+			Assert.fail("It should validate successfully");
+		}
```

Comment:
This try-catch is not necessary. You could just call `.validate`, if it fails the test fails, if not the test passes. There a bunch of tests like that, so it's safe to do it